db/seeds.rb

```diff
@@ -122,3 +122,5 @@
 
 # Related content
 Setting['related_content_score_threshold'] = -0.3
+
+Setting["feature.user.override_verification"] = nil
```

Comment:
As a default, better to set skip verification to `true` 
People need time to configure their verification system, 
and it's a common request to skip verification in the initial setup